kitsune/search/v2/documents.py

```diff
@@ -93,7 +93,17 @@ def get_model(cls):
 
 
 class QuestionDocument(SumoDocument):
-    """"""
+    """
+    ES document for Questions. Every Question in DB gets a QuestionDocument in ES.
+
+    Parent class to AnswerDocument, so most fields are prefixed with "question_".
+    QuestionDocument and AnswerDocument are stored in the same index, so QuestionDocument
+    cannot define a field with the same name as AnswerDocument.
+    Distinguish QuestionDocuments from AnswerDocuments by filtering on whether the
+    document has a null "created" field.
```

Comment:
I wouldn't add how to distinguish between the two. I would talk more about the arch - denormalized version of the AAQ for ES and about the fact that `QuestionDocument` is the parent class of AnswerDocument but in the end it's a single document.